p1pop.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Dobra praktyka. Brakuje atrybutu `lang` tj: `lang="pl-PL"` jest to deklaracja języka stosowanego na naszej stronie internetowej